<?xml version="1.0"?>

<launch>

<node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" respawn="false" output="screen"/>
	

<param name="robot_description" command="$(find xacro)/xacro.py '$(find armadillo_urdf)/robots/armadillo.xacro'" />


	<node name="armadillo2_hardware_interface" pkg="robotican_hardware_interface" type="armadillo2_hardware_interface_node" respawn="false" output="screen" />


  

    <rosparam file="$(find robotican_hardware_interface)/config/arm_config.yaml" command="load"/>

  <rosparam file="$(find robotican_hardware_interface)/config/controllers.yaml" command="load"/>

	<node name="arm_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="arm_trajectory_controller gripper_controller robot_state_controller pan_tilt_trajectory_controller mobile_base_controller torso_controller" />

<include file="$(find armadillo2_moveit_config)/launch/move_group.launch" />

<node
    name="map_base_tf"
    pkg="tf"
    type="static_transform_publisher"
    args="0 0 0 0 0 0 map odom 100" />
 


  <rosparam file="$(find robotican_hardware_interface)/config/hardware_interface_config.yaml" command="load"/>


</launch>
